pkg/server/container_create.go

```diff
@@ -513,6 +513,13 @@ func addOCIBindMounts(g *generate.Generator, mounts []*runtime.Mount, mountLabel
 			glog.Warningf("Unknown propagation mode for hostPath %q", mount.HostPath)
 			options = append(options, "rprivate")
 		}
+                // Set the rootfs propogation to rshared (recursive shared) for runc in the spec 
```

Comment:
We should default root propagation to `rprivate`.

If there is `rslave` bind mount, and rootfspropagation is `rprivate`, then update it to `rslave`.
If there is `rshared` bind mount, and rootfspropagation is not `rshared`, then update it to `rshared`.

And we should put them into the cases.

Reply:
default root propagation is rprivate in runc. Will update the rslave case